test/unit/explain.test.ts

```diff
@@ -0,0 +1,53 @@
+import { expect } from 'chai';
+import { it, test } from 'mocha';
+
+import { Explain, ExplainVerbosity } from '../mongodb';
+
+describe('class Explain {}', function () {
+  describe('static .fromOptions()', function () {
+    test('when no options are provided, it returns undefined', function () {
```

Comment:
Why the move to using `mocha.test` instead of `mocha.it`. I know they're aliases for each other, but we typically stick with the [BDD patterns](https://mochajs.org/#bdd) as opposed to mixing with the [TDD patterns](https://mochajs.org/#tdd).

Reply:
I usually use test when I specify both the context and behavior in one string because "it when no options are provided, it returns undefined" doesn't sound right.  I can change these if you'd like though

Reply:
a practical reason to stick with `it` is our lint rule won't work on this block

https://github.com/lo1tuma/eslint-plugin-mocha/blob/main/lib/rules/consistent-spacing-between-blocks.js#L4-L11

Reply:
Oh, good point.  

I adjusted both files to use `it`.